subsys/bluetooth/controller/ll_sw/ull_adv_iso.c

```diff
@@ -67,7 +71,8 @@ uint8_t ll_big_create(uint8_t big_handle, uint8_t adv_handle, uint8_t num_bis,
 	 * the periodic advertising trains is already associated
 	 * with another BIG.
 	 */
-	if (!adv || !adv->lll.sync || adv->lll.sync->adv_iso) {
+	lll_adv_sync = adv->lll.sync;
+	if (!adv || !lll_adv_sync || lll_adv_sync->iso) {
```

Comment:
I think you'd need the check for `adv` before accessing the members (`adv->lll.sync`)

Reply:
As the if clause will short-circuit on `!adv` the assignment `lll_adv_sync = adv->lll.sync;` is harmless (only assigning the equivalent similar to an `offsetof`)

Reply:
But if `adv` is NULL here, then wouldn't `lll_adv_sync = adv->lll.sync;` fail?

Reply:
No. It would not fail. But I will change to have explicit if checks to avoid the load operation.